services/UserService.js

```diff
@@ -1,15 +1,20 @@
 export default class UserService {
-    constructor(userModel, Op) {
+    constructor(userModel, Op, groupModel) {
         this.userModel = userModel;
         this.Op = Op;
+        this.groupModel = groupModel;
     }
 
     async findAll() {
-        return await this.userModel.findAll();
+        return await this.userModel.findAll({
+            include: this.groupModel
+        });
     }
 
     async findById(userId) {
-        return await this.userModel.findByPk(userId);
+        return await this.userModel.findByPk(userId, {
+            include: this.groupModel
+        });
```

Comment:
It is not a part of our task. I guess that you've done it to make the result easier to see and it's not a mistake at all. Just want to highlight that in case if you need to show or get some results from several tables it's better to create a new table with all the required fields. With current approach performance of our app will lower because each time the database will have to calculate all the data you need.

Reply:
Yes, I just added it to verify that mapping between User and Groups  work as expected and I can see related user/group entities. Thank you for valuable note about it, I will keep it in mind